packages/rolldown/tests/fixtures/topics/preserve-modules/cjs-inter-op/_config.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'vitest'
+import { defineTest } from 'rolldown-tests'
+import {OutputChunk} from 'rolldown'
+
+
+export default defineTest({
+  config: {
+    output: {
+      preserveModules: true
+    }
+  },
+  afterTest: (output) => {
+    expect(output.output[0].fileName).toMatchInlineSnapshot(`"main.js"`);
+    expect(output.output[0].code).toMatchInlineSnapshot(`
```

Comment:
Seems more proper to put it in the rust side if you just want to do snapshot test.

Reply:
Adding this test case to make sure the cjs module is correctly generated, and check its fs structure.I remove the other snapshot.